.github/workflows/check_docs.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v29.0.4
+        uses: tj-actions/changed-files@v41.0.0
```

Comment:
Let's remove this as we will use a different docs check this year

Reply:
removed and added note about removal to the changelog